.github/workflows/build-test-deploy.yml

```diff
@@ -558,19 +558,17 @@ jobs:
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
           name: composable-community
       - run: |
-          cachix watch-exec -j 16 -c 0 composable-community nix -- build .#kusama-picasso-karura-devnet --no-update-lock-file --show-trace -L
+          cachix watch-exec --jobs 16 --compression-level 0 composable-community nix -- build .#kusama-picasso-karura-devnet --no-update-lock-file --show-trace -L
 
-  package-xcvm:
-    name: "Package XCVM"
-    runs-on:
-      - self-hosted
-      - linux
-      - x64
-      - sre
+  package-xcvm-minimal:
+    name: "Package XCVM minimal"
+    runs-on: ubuntu-latest
     concurrency:
       group: ${{ github.workflow }}-package-xcvm-${{ github.ref }}
       cancel-in-progress: true
     continue-on-error: true
+    env:
+      COMPRESSION_LEVEL: ${{ github.event_name == 'push' && github.ref_name == 'main' && 1 || 0 }}
```

Comment:
```
steps:
# ...
  - if: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
    run: |
      echo "COMPRESSION_LEVEL=1" >> $GITHUB_ENV
  - if: ${{ github.event_name == 'push' && github.ref_name != 'main' }}
    run: |
      echo "COMPRESSION_LEVEL=0" >> $GITHUB_ENV  
# ...
```
?

Reply:
I think env is better, because:
- it is one liner. i really like one liners - easer to cp paste, easier to read from left to right. no need to go down. and less copy paste of var names COMPRESSION_LEVEL named only once.  may be will rename to CACIX_CO.._LE..
- the event and branch are available in scope of workflow, so may be will put it into scope of whole workflow, not only job